Jenkinsfile

```diff
@@ -0,0 +1,89 @@
+pipeline {
+    agent { 
+        docker { 
+            image 'argo.registry:5000/debian-jessie-argodoc' 
+        }
+    }
+    options {
+        checkoutToSubdirectory('argodoc')
+        newContainerPerStage()
+    }
+    environment {
+        RPOJECT_NAME="argodoc"
+    }
+    stages {
+        stage('Build') {
+            parallel {
+                stage ('Get web-api docs'){
+                    steps {
+                        dir ("${WORKSPACE}/argo-web-api") {
+                            git branch: 'devel',
+                                credentialsId: 'jenkins-rpm-repo',
+                                url: 'git@github.com:ARGOeu/argo-web-api.git'
+                            sh """
+                                cd ${WORKSPACE}/argo-web-api/doc/v2
+                                mkdocs build --clean
+                                cp -R ${WORKSPACE}/argo-web-api/doc/doc/ ${WORKSPACE}/argodoc/content/guides
```

Comment:
There are changes in the scripts bellow like: 
- in messaging and in auth the api version
- in api authn the path is "docs" instead for "doc"

So the variable will not solve any of this problems instead it will have an impact in the readability of the script